packages/components/src/components/telekom/app-footer/app-footer.tsx

```diff
@@ -60,7 +63,16 @@ export class AppFooter {
                 {this.hasSlotLogo ? (
                   <slot name="logo"></slot>
                 ) : (
-                  <app-logo claim claimLang={this.claimLang}></app-logo>
+                  <scale-logo
+                    transparent
+                    variant="white"
+                    language=""
+                    size={24}
+                    href={this.logoHref}
+                    logoTitle={this.logoTitle}
+                    onClick={this.logoClick}
+                    ariaDescribedBy={this.logoAriaDescribedBy}
```

Comment:
Shouldn't this be `logoAriaDescribedBy={this.logoAriaDescribedBy}`?

Reply:
ah thanks! <3